### ISSUE TITLE
[ SUPPORT REQUEST ] - Natsuki is not talking first

### ISSUE THREAD

**Issue by FastestDan** [Just-Natsuki-Team/NatsukiModDev#165]
# The Problem:
I waited for 15 minutes for Natsuki to start a conversation, but she didn't do anything except gazing from left to right. I then set the "random conversation" setting (I believe it's called something like that) to the max value (all the way to the right) and left the game running for about an hour and a half. When I checked, the only difference was the outfit she was in. (She was in a school uniform and changed it to her casual clothes). Is it something that I do wrong or is it something wrong with the game or, hell, RenPy itself?

**Comment by Blizzardsev** [comment]
Hiya,

Thanks for bringing this up to us. I'll take a look and see if I can replicate the problem.

**Comment by Blizzardsev** [comment]
Thank you for your patience - it looks like this was related to a UI change we made some time ago. In short, we adjusted the scaling of the talk frequency to increase in level from left to right, but missed a change to the code behind that which actually determined the pause between talks.

I'm putting a fix together now - this should be in at some point today.

I'll update again when this is resolved.
